common/mixins/utils.js

```diff
@@ -5,15 +5,16 @@ var Promise = require('bluebird');
 
 var path = require('path');
 var fileName = path.basename(__filename, '.js'); // gives the filename without the .js extension
-var log = require('./../lib/debug-extension')('common:models:'+fileName);
+const logger = require('sp-json-logger');
 
 module.exports = function(Model, options) {
 
   Model.getCurrentUserModel = function(cb) {
     var ctx = loopback.getCurrentContext();
     var currentUser = ctx && ctx.get('currentUser');
     if (currentUser) {
-      log.trace('inside ' + Model.definition.name + '.getCurrentUserModel() - currentUser: ', currentUser.username);
+      //log.trace('inside ' + Model.definition.name + '.getCurrentUserModel() - currentUser: ', currentUser.username);
+      logger.trace({log: {message: `inside ${Model.definition.name}.getCurrentUserModel() - currentUser: ${currentUser.username}` }});
```

Comment:
Don't you think writing everything under `log.message` is a bit exhausting for all logs? 
There could be a utility function that could do this job on itself.
Something that would change:
`logger.trace('inside ${Model.definition.name}.getCurrentUserModel() - currentUser: ${currentUser.username}'); `
TO
`logger.trace({log: {message: 'inside ${Model.definition.name}.getCurrentUserModel() - currentUser: ${currentUser.username}' }});`

Reply:
So maybe when only one string argument is available, the logger can automap it to `{log: {message: <arg1>} }` and the rest of the times, we will use the object structure when writing the logs as developers ... thoughts?

Reply:
`logger.<level>('my ' + arg1 + ' should be printed with '  + JSON.stringify(arg2));`
becomes:
`logger.<level>( { log: { message: 'my ' + arg1 + ' should be printed with '  + JSON.stringify(arg2) } } );`

Reply:
and when its important to log differently, then the developers can choose to write themselves:
`logger.<level>( { log: { message: 'some reasoning',  arg1: arg1, arg2: arg2 } } );`

Reply:
So maybe when only one string argument is available, the logger can automap it to {log: ```{message: <arg1>} }``` +1

```logger.<level>( { log: { message: 'my ' + arg1 + ' should be printed with ' + JSON.stringify(arg2) } } );``` 
Second example might create problem in logstash pipeline while parsing to json, and we will loose ability to index individual fields if we are stringifying json object inside message field! Should we sacrifice that for ease of use?